hiclass/LocalClassifierPerParentNode.py

```diff
@@ -225,14 +225,18 @@ def _fit_digraph(self, local_mode: bool = False, use_joblib: bool = False):
         nodes = self._get_parents()
         self._fit_node_classifier(nodes, local_mode, use_joblib)
 
-    def _get_predict_proba(model, X):
-        classifiers = [
-            model.hierarchy_.nodes[node]["classifier"]
-            for node in model.hierarchy_.nodes
-            if "classifier" in model.hierarchy_.nodes[node]
+    def get_predict_proba(self, X):
```

Comment:
@jannikgro I have updated the function to return a dict, however it currently just returns predict_proba for all nodes in the hierarchy. I'm woking on another version which only gives the dict for only traversed nodes.